subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/SimpleGlobalFilesBuildSettingsDescriptor.java

```diff
@@ -53,6 +53,14 @@ private BuildScriptBuilder builder(InitSettings settings) {
         if (!settings.getSubprojects().isEmpty()) {
             builder.methodInvocation(null, "include", settings.getSubprojects().toArray());
         }
+
+        if(settings.getJavaLanguageVersion().isPresent()){
+            builder.plugin(
+                "Apply the foojay plugin to allow automatic download of matching JDKs on other systems",
+                "org.gradle.toolchains.foojay-resolver",
+                "0.4.0");
+        }
+
```

Comment:
:lipstick: I would put this after `pluginManagement {}` and before `rootProject.name` so that the code follows the order of the generated settings script more closely, for clarity